Library/Homebrew/.simplecov

```diff
@@ -38,11 +38,22 @@ SimpleCov.start do
     track_files "#{SimpleCov.root}/{#{subdirs},*.rb}"
   end
 
+  add_filter %r{^/build.rb$}
+  add_filter %r{^/config.rb$}
+  add_filter %r{^/constants.rb$}
```

Comment:
Shouldn‘t this be covered by running any integration test?

Reply:
Seems not.

Reply:
Or it's been already included.